ethcore/src/account_provider/mod.rs

```diff
@@ -280,16 +296,16 @@ impl AccountProvider {
 	pub fn accounts(&self) -> Result<Vec<Address>, Error> {
 		let accounts = self.sstore.accounts()?;
 		Ok(accounts
-		   .into_iter()
-		   .map(|a| a.address)
-		   .filter(|address| !self.blacklisted_accounts.contains(address))
-		   .collect()
+			.into_iter()
+			.map(|a| a.address)
+			.filter(|address| !self.blacklisted_accounts.contains(address))
+			.collect()
 		)
 	}
 
 	/// Returns addresses of hardware accounts.
 	pub fn hardware_accounts(&self) -> Result<Vec<Address>, Error> {
```

Comment:
How about making this private and return `Error`? Seems like the only caller is `hardware_accounts_info()` and it a little misleading as `hardware_accounts()` doesn't actually ever fail (due to the `map_or_else`)?

Reply:
Yeah, that is a great suggestion i.e, to return an `Error` but it can't be private because `rpc` actually calls this!